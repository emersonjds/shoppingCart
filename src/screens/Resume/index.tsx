import React, { useEffect } from 'react';
import { View } from 'react-native';
import { useSelector } from 'react-redux';
import { shoppingSelector } from '../../redux/slices/shoppingSlice';

// import { Container } from './styles';

const Resume: React.FC = () => {
  const { cart } = useSelector(shoppingSelector);

  useEffect(() => {
    console.log('DADOS', cart);
  }, []);

  return <View testID="Resume" />;
};

export default Resume;
