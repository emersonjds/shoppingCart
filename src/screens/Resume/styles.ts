import { Text } from 'react-native';
import styled from 'styled-components/native';
import Colors from '../../contants';

export const Title = styled.Text`
  color: ${Colors.PRIMARY_TEXT_COLOR};
`;

export const ButtonQuantity = styled.TouchableOpacity`
  height: 25px;
  width: 25px;
  border-radius: 20px;
  background-color: ${Colors.PRIMARY_COLOR};
  align-items: center;
  justify-content: center;
`;

export const ButtonQuantityText = styled.Text`
  color: ${Colors.SECONDARY_BACKGROUND_COLOR};
  font-size: 14px;
`;

export const TextNavigate = styled.Text`
  text-align: center;
  font-weight: bold;
  font-size: 16px;
  color: ${Colors.PRIMARY_COLOR};
`;

export const TextTotal = styled.Text`
  font-size: 16px;
  color: ${Colors.PRIMARY_TEXT_COLOR};
  font-weight: bold;
`;

export const TextValue = styled.Text`
  font-size: 25px;
  font-wereight: bold;
  color: ${Colors.PRIMARY_COLOR};
`;

export const ButtonFinalizePurchase = styled.Text`
  height: 52px;
  width: 80%;
  background-color: ${Colors.PRIMARY_COLOR};
  align-items: center;
  justify-content: center;
  border-radius: 50px;
`;

export const TextButtonPurchase = styled.Text`
  color: ${Colors.SECONDARY_BACKGROUND_COLOR};
  font-size: 20px;
  font-weight: bold;
`;
