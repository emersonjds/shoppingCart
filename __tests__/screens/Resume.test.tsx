import React from 'react';
import { render, cleanup } from '@testing-library/react-native';
import Resume from '../../src/screens/Resume';

jest.mock('../../src/screens/Resume', () => 'Resume');

afterEach(cleanup);

describe('Resume', () => {
  it('renders correctly', () => {
    const wrapper = render(<Resume />);
    expect(wrapper).toMatchSnapshot();
  });
});
