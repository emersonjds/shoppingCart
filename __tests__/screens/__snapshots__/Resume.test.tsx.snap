// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Resume renders correctly 1`] = `<Resume />`;
